examples/structured_2d_dgsem/elixir_eulerpolytropic_wave.jl

```diff
@@ -0,0 +1,88 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the polytropic Euler equations
+
+gamma = 1.001   # Quasi-isotropic simulation. gamma = 1.0 would lead to a singularity.
+# gamma = 2.0   # Adiabatic monatomic gas in 2d.
+kappa = 1.0     # Scaling factor for the pressure.
+equations = PolytropicEulerEquations2D(gamma, kappa)
+
+# Linear pressure wave in the negative x-direction.
+function initial_condition_wave(x, t, equations::PolytropicEulerEquations2D)
```

Comment:
So is this a discontinuous test to check the EC property? If so the name of this initial condition, the name of the elixir, and some comments herein should be changed / added to have `ec` in the title and clarify what this test is doing.

Reply:
I'm testing the entropy conservation with `initial_condition_weak_blast_wave` in `examples/structured_2d_dgsem/elixir_eulerpolytropic_ec_shockcapturing.j`.  I'm getting `∑∂S/∂U ⋅ Uₜ :  -8.18482485e-05`.

Reply:
Hm, 1e-5 is _not_ EC, it should be rather something like 1e-15. Also, I believe the name of the elixir is slightly misleading, as it does not use shock capturing - maybe just drop the `_shockcapturing` bit, since it is intended as an EC test?